### PR TITLE
Simple CLI

### PR DESCRIPTION
**Pull request by andrewgryan**
This is long overdue, it allows for the following command line usages of FOREST

* `forest --show file.nc`
* `forest --show --file-type rdt rdt.json`
* `forest --config-file config.yml --database file.db`

This is a massive de-coupling that involved redesigning the architecture of the menu, there are usability concerns that I will tackle in another pull request. Since the existing UI is not fit for the new use cases

### CODE REVIEW

**Comment by andrewgryan** [comment]
It also adds a `forest.sample.build_all()` method to build sample files to test the command line interface manually

**Comment by andrewgryan** [comment]
Some documentation might be needed to explain the new usage and to improve the `forest.sample` facility

**Comment by andrewgryan** [comment]
Also, the new UI needs to populate the first dropdown

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 bokeh
-scitools-iris
+iris
```
Comment (bjlittle) [inline requirements.txt]: This is the `conda` package name, not the PyPI name... just making sure that's what you want, right?

Comment (andrewgryan) [reply requirements.txt]: Yes, I think I changed it to `scitools-iris` to get the docs building but it broke the Travis checks so I changed it back and added `doc/requirements.txt` to keep a trimmed list for readthedocs integration

**Pull request merged**